src/components/index.cjs

```diff
@@ -0,0 +1,16 @@
+const paths = require.context('./', true, /\.js$/).keys();
```

Comment:
@domchrz why do you use here cjs? Could we just re-export the components statically here?

Reply:
Yes, that works, I just thought it would be redundant, since we write the above once and it will add any component in the folder automatically. Should I change it?

Reply:
@domchrz Yeah, It's quite common to have static re-export in React project since it doesn't bring significant complexity in most cases. Also, I think it will be clearer here, considering that your logic involves is that looks inconsistent with the rest of the codebase.

Reply:
@MakaryHryshkin done!